### ISSUE TITLE
Blank screen when opening RESX Resource Manager

### ISSUE THREAD

**Issue by iVcardo** [tom-englert/ResXResourceManager#98]
Since updating VS 2017 today RESX Resource Manager is not working, just a blank screen?? I have been using this product successfully for over a year now without a problem. By the way the solution is open just in case you wondered. I have uninstalled and re-installed but still not joy.

![image](https://user-images.githubusercontent.com/29012424/37307277-af44f26a-263a-11e8-9308-f63f75ab81e5.png)

**Comment by iVcardo** [comment]
Just to add to the above I have been using this product successfully for over a year now without a problem. By the way the solution is open just in case you wondered.

**Comment by bigshawn1478** [comment]
Same problem on VS 2015
![default](https://user-images.githubusercontent.com/12092499/37321449-60155ca6-26b3-11e8-8450-bc1239134cb4.JPG)

ResX version: 1.12.976

Here is the error message from Output window
```
ResX Resource Manager loaded.
Home: https://github.com/tom-englert/ResXResourceManager
Report issues: https://github.com/tom-englert/ResXResourceManager/issues
Support the project by adding a short review: https://marketplace.visualstudio.com/items?itemName=TomEnglert.ResXManager#review-details
Assembly location: C:\Users\shawn\AppData\Local\Microsoft\VisualStudio\14.0\Extensions\obmgdcqr.v3y
Version: 1.12.976.0
Error: System.ComponentModel.Composition.CompositionException: The composition produced a single composition error. The root cause is provided below. Review the CompositionException.Errors property for more detailed information.

1) Could not load file or assembly 'Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. 系統找不到指定的檔案。

Resulting in: An exception occurred while trying to create an instance of type 'tomenglertde.ResXManager.Translators.TranslatorHost'.

Resulting in: Cannot activate part 'tomenglertde.ResXManager.Translators.TranslatorHost'.
Element: tomenglertde.ResXManager.Translators.TranslatorHost -->  tomenglertde.ResXManager.Translators.TranslatorHost -->  AssemblyCatalog (Assembly="ResXManager.Translators, Version=1.12.976.0, Culture=neutral, PublicKeyToken=695a75ddb69d5a4b")

Resulting in: Cannot get export 'tomenglertde.ResXManager.Translators.TranslatorHost (ContractName="tomenglertde.ResXManager.Translators.TranslatorHost")' from part 'tomenglertde.ResXManager.Translators.TranslatorHost'.
Element: tomenglertde.ResXManager.Translators.TranslatorHost (ContractName="tomenglertde.ResXManager.Translators.TranslatorHost") -->  tomenglertde.ResXManager.Translators.TranslatorHost -->  AssemblyCatalog (Assembly="ResXManager.Translators, Version=1.12.976.0, Culture=neutral, PublicKeyToken=695a75ddb69d5a4b")

Resulting in: Cannot set import 'tomenglertde.ResXManager.View.Visuals.TranslationsViewModel..ctor (Parameter="translatorHost", ContractName="tomenglertde.ResXManager.Translators.TranslatorHost")' on part 'tomenglertde.ResXManager.View.Visuals.TranslationsViewModel'.
Element: tomenglertde.ResXManager.View.Visuals.TranslationsViewModel..ctor (Parameter="translatorHost", ContractName="tomenglertde.ResXManager.Translators.TranslatorHost") -->  tomenglertde.ResXManager.View.Visuals.TranslationsViewModel -->  AssemblyCatalog (Assembly="ResXManager.View, Version=1.12.976.0, Culture=neutral, PublicKeyToken=695a75ddb69d5a4b")

Resulting in: Cannot get export 'tomenglertde.ResXManager.View.Visuals.TranslationsViewModel (ContractName="VisualComposition-86E8D1EF-1322-46B4-905C-115AAD63533D")' from part 'tomenglertde.ResXManager.View.Visuals.TranslationsViewModel'.
Element: tomenglertde.ResXManager.View.Visuals.TranslationsViewModel (ContractName="VisualComposition-86E8D1EF-1322-46B4-905C-115AAD63533D") -->  tomenglertde.ResXManager.View.Visuals.TranslationsViewModel -->  AssemblyCatalog (Assembly="ResXManager.View, Version=1.12.976.0, Culture=neutral, PublicKeyToken=695a75ddb69d5a4b")

   at System.ComponentModel.Composition.Hosting.CompositionServices.GetExportedValueFromComposedPart(ImportEngine engine, ComposablePart part, ExportDefinition definition)
   at System.ComponentModel.Composition.Hosting.CatalogExportProvider.GetExportedValue(CatalogPart part, ExportDefinition export, Boolean isSharedPart)
   at System.ComponentModel.Composition.Hosting.CatalogExportProvider.CatalogExport.GetExportedValueCore()
   at System.ComponentModel.Composition.Primitives.Export.get_Value()
   at System.ComponentModel.Composition.ExportServices.GetCastedExportedValue[T](Export export)
   at System.ComponentModel.Composition.ExportServices.<>c__DisplayClass11_0`2.<CreateStronglyTypedLazyOfTM>b__1()
   at System.Lazy`1.CreateValue()
   at System.Lazy`1.LazyInitValue()
   at System.Lazy`1.get_Value()
   at TomsToolbox.Wpf.Composition.ItemsControlCompositionBehavior.<OnUpdate>b__4_1(Lazy`2 item)
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at TomsToolbox.Wpf.Composition.ItemsControlCompositionBehavior.OnUpdate()
   at TomsToolbox.Wpf.Composition.VisualCompositionBehavior`1.Update()
```

**Comment by rama21385** [comment]
I can confirm the problem: VS 15.6.1 Professional

**Comment by GeoRFID** [comment]
Exactly the same on VS Professional 2015 (Version 14.0.25431.01 Update 3)

**Comment by vanengelen** [comment]
As a quick fix i copied the Newtonsoft.Json.dll file from an other project in the folder mentioned in the error message. In my case c:\Users\erik\AppData\Local\Microsoft\VisualStudio\14.0\Extensions\mxiin3v3.hu0. This seems to work for now.

**Comment by GeoRFID** [comment]
I think it is a problem of version 1.12 of ResXResource Manager, uninstaling it and manually install 1.11 works again

**Comment by iVcardo** [comment]
I can confirm it is a V 1.12 version issue of ResX Resource Manager, uninstalling it and manually installing 1.11 works, then disable automatic updates.

**Comment by ComputeriseIt** [comment]
Same here in Visual Studio Community 15.6.1

**Comment by larwine** [comment]
Same here in Community 15
I had to uninstall the new version, download the old version and install for it to start working again

**Comment by eduardoalba** [comment]
I can confirm the problem: VS 15

**Comment by ningin** [comment]
vanengelen's solution is work,thank you!

**Comment by hilk** [comment]
I can confirm too in vs 15 pro.  iVcardo's solution work for me and i disable automatic updates.

**Comment by iVcardo** [comment]
@tom-englert any chance of a fix for this issue Tom? Thanks

**Comment by tofupl** [comment]
I can confirm too in vs 15 pro.

**Comment by MaximSaraev** [comment]
The same for VS 2015 Enterprise

**Comment by tom-englert** [comment]
Fixed in V1.13
